src/frontend/js/utils/StringHelper/index.spec.tsx

```diff
@@ -26,4 +31,9 @@ describe('StringHelper', () => {
     expect(StringHelper.capitalizeFirst(null)).toBe(null);
     expect(StringHelper.capitalizeFirst(undefined)).toBe(undefined);
   });
+
+  it('abbreviates a string', () => {
+    expect(StringHelper.abbreviate('John Doe')).toBe('JD');
+    expect(StringHelper.abbreviate('John Doe', 1)).toBe('J');
+  });
```

Comment:
```js
expect(expect(StringHelper.abbreviate('john doe')toBe('JD');
expect(expect(StringHelper.abbreviate('Jean-Charle Van Dam')).toBe('JCVD');
expect(expect(StringHelper.abbreviate('Jean-charle Van dam')).toBe('JCVD');
```

Reply:
Those tests will fail. As this is an abbreviation, indeed we could assume to uppercase chars.

But about split strings by dash, I'm not quit sure 🤔 

```
expect(expect(StringHelper.abbreviate('john doe')toBe('JD'); // will return 'jd'
expect(expect(StringHelper.abbreviate('Jean-Charle Van Dam')).toBe('JCVD'); // will return 'JVD'
expect(expect(StringHelper.abbreviate('Jean-charle Van dam')).toBe('JCVD'); // will return 'JVd`
```